features/functests/performance/performance.go

```diff
@@ -1,73 +1,214 @@
+			_, err := clients.MachineConfig.MachineconfigurationV1().MachineConfigs().Get(perfWorkerPreBootTuning, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred(), "doesn't contain Machine Configuration profile "+perfWorkerPreBootTuning)
+		})
+
+		const perfRtKernelPrebootTuningScript = "/usr/local/bin/pre-boot-tuning.sh"
+		It(perfRtKernelPrebootTuningScript+" should exist on the nodes", func() {
+			nodes := getListOfNodes()
+			checkWorkerRtProfileReadiness(nodes)
+			checkFileExistense(nodes, perfRtKernelPrebootTuningScript)
+		})
+
+		It("Should contain a custom initrd image in the boot loader", func() {
+			nodes := getListOfNodes()
+			checkWorkerRtProfileReadiness(nodes)
+			for _, node := range nodes {
+				mcd, err := mcdForNode(&node)
+				Expect(err).ToNot(HaveOccurred())
+				mcdName := mcd.ObjectMeta.Name
+				By("executing the command \"grep -R  initrd /rootfs/boot/loader/entries/\" inside the pod " + mcdName)
+				bootLoaderEntries, err := exec.Command("oc", "rsh", "-n", mcd.ObjectMeta.Namespace, "-c",
+					perfMachineConfigDaemonContainer, mcdName, "grep", "-R", "initrd", "/rootfs/boot/loader/entries/").CombinedOutput()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(strings.Contains(string(bootLoaderEntries), "iso_initrd.img")).To(BeTrue(), "cannot find iso_initrd.img entry among the bootloader entries")
+			}
+		})
+	})
+
+	// 11-machine-config-worker-rt-kernel.yaml related verifications
+	var _ = Context("Cluster configuration", func() {
+		const perfWorkerRTKernelMC = "11-worker-rt-kernel"
+		It("Should contain Machine Configuration profile "+perfWorkerRTKernelMC, func() {
+			_, err := clients.MachineConfig.MachineconfigurationV1().MachineConfigs().Get(perfWorkerRTKernelMC, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred(), "doesn't contain Machine Configuration profile "+perfWorkerRTKernelMC)
+		})
+
+		const perfRTKernelPatchScript = "/usr/local/bin/rt-kernel-patch.sh"
+		It(perfRTKernelPatchScript+" should exist on the nodes", func() {
+			nodes := getListOfNodes()
+			checkWorkerRtProfileReadiness(nodes)
+			checkFileExistense(nodes, perfRTKernelPatchScript)
+		})
+
+		const perfKernelVersionValuePreemptEntry = "PREEMPT"
+		const perfKernelVersionValueRTEntry = "RT"
+		It(fmt.Sprintf("Should contain %s and %s entries",
+			perfKernelVersionValueRTEntry, perfKernelVersionValuePreemptEntry), func() {
+			nodes := getListOfNodes()
+			checkWorkerRtProfileReadiness(nodes)
+			for _, node := range nodes {
 				mcd, err := mcdForNode(&node)
 				Expect(err).ToNot(HaveOccurred())
 				mcdName := mcd.ObjectMeta.Name
-				kargsBytes, err := exec.Command("oc", "rsh", "-n", "openshift-machine-config-operator", mcdName,
+				By("executing the command \"uname -v\" inside the pod " + mcdName)
+				Eventually(func() bool {
+					out, err := exec.Command("oc", "rsh", "-n", mcd.ObjectMeta.Namespace,
+						"-c", perfMachineConfigDaemonContainer, mcdName, "uname", "-v").CombinedOutput()
```

Comment:
is there a chance we could miss the PREEMPT and RT values for using -v instead of -a ?
(for uname,  -a will cover everything)

Reply:
@yanirq , no uname -v works fine in our case